typescript-test-samples/apigw-lambda-sqs-sns-datadog/lib/apigw-lambda-sqs-sns-datadog-stack.ts

```diff
@@ -0,0 +1,63 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as sns from 'aws-cdk-lib/aws-sns';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as apigateway from 'aws-cdk-lib/aws-apigateway';
+import * as snsSubscriptions from 'aws-cdk-lib/aws-sns-subscriptions';
+import * as lambdaEventSources from 'aws-cdk-lib/aws-lambda-event-sources';
+import { Datadog } from "datadog-cdk-constructs-v2";
+
```

Comment:
Add import for `NodejsFunction` e.g.:
`import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';`